docs/_docs/02_features/auto-retry.md

```diff
@@ -24,10 +24,23 @@ Terraform can fail with transient errors which can be addressed by simply retryi
     - Checking for available provider plugins on https://releases.hashicorp.com...
     Error installing provider "template": error fetching checksums: Get https://releases.hashicorp.com/terraform-provider-template/1.0.0/terraform-provider-template_1.0.0_SHA256SUMS: net/http: TLS handshake timeout.
 
-Terragrunt sees this error, and knows it is a transient error that can addressed by re-running the `apply` command.
+Terragrunt sees this error, and knows it is a transient error that can be addressed by re-running the `apply` command.
 
 `auto-retry` will try a maximum of three times to re-run the command, at which point it will deem the error as not transient, and accept the terraform failure. Retries will occur when the error is encountered, pausing for 5 seconds between retries.
 
-Known errors that `auto-retry` will rerun, are maintained in the `TerragruntOptions.RetryableErrors` array. Future upgrades to `terragrunt` may include the ability to configure `auto-retry` by specifying additional error strings and configuring max retries and retry intervals the `terragrunt` config (PRs welcome\!).
+A list of default known errors that `auto-retry` will rerun is maintained in the `TerragruntOptions.RetryableErrors` array. These defaults can be overridden by specifying a list of retryable errors in the `terragrunt` config using the following format:
```

Comment:
NIT: Most users don't look at the Terragrunt source code, so they won't know what `TerragruntOptions.RetryableErrors` is. Here's a different phrasing:

```suggestion
Terragrunt has a small list of default known errors built-in. You can override these defaults with your own custom retryable errors in your `terragrunt.hcl` configuration:
```

Reply:
Makes total sense, I'll update.